src/frontend/src/lib/components/send/SendModal.svelte

```diff
@@ -25,7 +27,9 @@
 	let sendProgressStep: string = ProgressStepsSend.INITIALIZATION;
 
 	let steps: WizardSteps;
-	$: steps = sendWizardStepsComplete({ i18n: $i18n });
+	$: steps = isNullish($pageToken)
```

Comment:
Why `isNullish($pageToken)`?

Reply:
It is just to differentiate if we are in the token transactions page already (in this case skips the "Select token" step) or if any other page

Reply:
- I would suggest to not have a route related logic within this component. I would use a property and set that particular value in its consumer, in the here

- Instead of `isNullish($pageToken)` I would use an explicit logic such as `isRouteTokens`

Reply:
Have a look at the last commit please? Is that what you meant?

Reply:
Sounds good, thx